io/genbank/genbank.go

```diff
@@ -821,7 +785,7 @@ func buildMetaString(name string, data string) string {
 		if index == 0 {
 			returnData = name + datum + "\n"
 		} else {
-			returnData += generateWhiteSpace(11) + datum + "\n"
+			returnData += generateWhiteSpace(12) + datum + "\n"
```

Comment:
why the change to 12?

Reply:
Previously this behavior was just broken. See:

https://github.com/TimothyStiles/poly/blob/7f133c87716de08b873b44d5a405688da1ddebb4/data/puc19static.gbk#L10-L11

Reply:
Should be aligned